integration-tests/src/test/java/com/datastax/oss/driver/api/core/context/LifecycleListenerIT.java

```diff
@@ -75,7 +75,7 @@ public void should_not_notify_listener_when_init_fails() {
       simulacronRule.cluster().acceptConnections();
```

Comment:
in this test we have a logic:
```
  try (CqlSession session = newSession(listener)) {
      ConditionChecker.checkThat(() -> listener.ready).before(1, SECONDS).becomesTrue();
      assertThat(listener.closed).isFalse();
    }
    assertThat(listener.ready).isTrue();
    ConditionChecker.checkThat(() -> listener.closed).before(1, SECONDS).becomesTrue();
```
I am not sure why we need that check: ` assertThat(listener.ready).isTrue();`
after the try block. It is already validatied withing try block:
`ConditionChecker.checkThat(() -> listener.ready).before(1, SECONDS).becomesTrue();`
Maybe we can remove that second check?